Jint/Native/Json/JsonParser.cs

```diff
@@ -879,7 +876,7 @@ private class Extra
         {
             public int? Loc;
             public int[] Range;
-            public string Source;
+            public Script Source;
```

Comment:
Agreed the naming could be better.  Code and Source members of Script may be redundant.  They are used to track/compare breakpoints in required/included files.